src/LtiServiceConnector.php

```diff
@@ -56,10 +67,11 @@ public function get_access_token($scopes) {
         return $this->access_tokens[$scope_key] = $token_data['access_token'];
     }
 
-    public function make_service_request($scopes, $method, $url, $body = null, $content_type = 'application/json', $accept = 'application/json') {
+    public function makeServiceRequest($scopes, $method, $url, $body = null, $content_type = 'application/json', $accept = 'application/json')

```

Comment:
This function header does not match the interface, default params should be added to the interface and `$scopes` here should have an `array` type on it.